### PR TITLE
Show malformed JSON in case of missing rates key

### PR DESCRIPTION
**Pull request by vorburger**
@xuhcc merge?

### CODE REVIEW

File: beancount_exchangerates/source.py [hunk 1]
```diff
@@ -59,7 +59,10 @@ def _get_price(self, ticker, time=None):
             price_time = datetime.datetime.now(datetime.timezone.utc)
         else:
             result = json.loads(response.read().decode())
-            price = to_decimal(result['rates'][symbol])
+            try:
+                price = to_decimal(result['rates'][symbol])
+            except KeyError as err:
+                raise RuntimeError("There is no 'rates' key from " + url + " in: " + str(result))
```
Comment (xuhcc) [inline beancount_exchangerates/source.py]: KeyError may also occur if `<symbol>` key is missing.

I think the original KeyError message should be preserved.

Comment (vorburger) [reply beancount_exchangerates/source.py]: @xuhcc agreed, thanks for catching that. Better like this? (Note changes made, using "force push".)

Comment (xuhcc) [reply beancount_exchangerates/source.py]: No, I mean something like this:

```python
RuntimeError(f"KeyError: {str(err)}: {str(result)}")
```

`str(err)` would show what key is missing

Comment (vorburger) [reply beancount_exchangerates/source.py]: Oh, I see. Is this more "idiomatic" and typical, in Python? I'm new (to Python), and used to "chaining" exceptions, from Java! 😄 

I've now updated it as (I think) you suggested. Better like this?

Comment (xuhcc) [reply beancount_exchangerates/source.py]: Yes, that's better. Thanks

Regarding chaining, I don't know if it's more typical, I'd fine with `raise ... from` too.

**Pull request merged**